content/id/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,702 @@
+|   Block       | Filesystem      | TIDAK TERIKAT    |
+|   Filesystem  | Filesystem      | TERIKAT          |
+|   Filesystem  | Block           | TIDAK TERIKAT    |
+|   Filesystem  | unspecified     | TERIKAT          |
+
+{{< note >}}
+Hanya volume yang disediakan secara statis yang didukung untuk rilis alfa. Administrator harus memperhatikan nilai-nilai tersebut ketika mengerjakan perangkat-perangkat _raw block_.
+{{< /note >}}
+
+## _Volume Snapshot_ dan Dukungan Pemulihan Volume dari _Snapshot_
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Fitur _volume snapshot_ ditambahkan hanya untuk mendukung CSI Volume Plugins. Untuk lebih detail, lihat [_volume snapshots_](/docs/concepts/storage/volume-snapshots/).
+
+Untuk mengaktifkan dukungan pemulihan sebuah volume dari sebuah sumber data _volume snapshot_, aktifkan
+gerbang fitur `VolumeSnapshotDataSource` pada apiserver dan _controller-manager_.
+
+### Membuat _Persistent Volume Claim_ dari _Volume Snapshot_
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: restore-pvc
+spec:
+  storageClassName: csi-hostpath-sc
+  dataSource:
+    name: new-snapshot-test
+    kind: VolumeSnapshot
+    apiGroup: snapshot.storage.k8s.io
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+```
+
+## Menulis Konfigurasi Portabel
+
+Jika kamu menulis templat konfigurasi atau contoh yang dapat berjalan pada berbagai macam kluster
+dan membutuhkan _persistent storage_, kami merekomendasikan agar kamu menggunakan pola berikut:
+
+- Masukkan objek PersistentVolumeClaim pada kumpulan _config_ (bersamaan dengan
+  Deployments, ConfigMaps, dsb).
+- Jangan memasukkan objek PersistentVolume pada _config_, karena pengguna yang menginstantiasi
+  _config_ tersebut kemungkinan tidak memiliki izin untuk membuat PersistentVolume.
+- Berikan pengguna opsi untuk menyediakan nama _storage class_ ketika menginstantiasi
+  templat.
+  - Jika pengguna menyediakan nama _storage class_, taruh nilai tersebut pada
+    kolom `persistentVolumeClaim.storageClassName`.
+    Hal ini akan membuat PVC agar sesuai dengan _storage class_
+    yang tepat jika kluster memiliki banyak StorageClass yang diaktifkan oleh admin.
+  - Jika pengguna tidak menyediakan nama _storage class_, biarkan
+    kolom `persistentVolumeClaim.storageClassName` kosong.
+    - Hal ini kakan membuat sebuah PV disediakan secara otomatis untuk pengguna dengan
+      StorageClass standar pada kluster.  Banyak lingkungan kluster memiliki
+      StorageClass standar yang sudah terpasang, atau administrator dapat membuat
+      StorageClass standar sendiri.
+- Dalam pembuatan, perhatikan PVC yang tidak kunjung terikat setelah beberapa lama
+  dan beritahukan hal ini pada pengguna, karena hal ini dapat mengindikasikan kluster tidak
+  memiliki dukungan storage dinamis (di mana pengguna harus membuat PV yang sesuai)
+  atau kluster tidak memiliki sistem storage (di mana penggun tidak dapat membuat
```

Comment:
please make it consistent whether using _storage_ or penyimpanan

Reply:
okay! resolved in https://github.com/kubernetes/website/pull/14532/commits/9292f151c434f101706b1290e2ecb3e04630b3e5